src/packagedcode/rubygems.py

```diff
@@ -385,6 +398,22 @@ def build_rubygem_package(gem_data, download_url=None, package_url=None):
     return package
 
 
+def licenses_mapper(license, licenses, package):
+    """
+    Update package licensing and return package based on the `license` and
+    `licenses` values found in a package.
+    """
+    declared_licenses = []
+    if license:
+        declared_licenses.append(str(license).strip())
+    if licenses:
+        for lic in licenses:
+            if lic and lic.strip():
+                declared_licenses.append(lic.strip())
+    package.declared_license = declared_licenses
```

Comment:
Instead of updating the package, please return the list of declared licenses.

Reply:
Fixed.